ui/app/AppLayouts/Wallet/services/dapps/DappsConnectorSDK.qml

```diff
@@ -334,7 +334,7 @@ WalletConnectSDKBase {
                             url
                         }
                     },
-                    requiredNamespaces: supportedNamespaces
+                    requiredNamespaces: JSON.parse(supportedNamespaces)
```

Comment:
Intended?

Reply:
yes. We have a string and we'll need an object.